### PR TITLE
Everywhere: Fix more typos

### PR DESCRIPTION
**Pull request by linusg**
Hate to do this in October (:shirt:) but @nico introduced me to `codespell` so I had to.

No idea why so many are left even after #3662?!

### CODE REVIEW

File: Base/etc/services [hunk 1]
```diff
@@ -392,7 +392,7 @@ afs3-vlserver	7003/tcp			# volume location database
 afs3-vlserver	7003/udp
 afs3-kaserver	7004/tcp			# AFS/Kerberos authentication
 afs3-kaserver	7004/udp
-afs3-volser	7005/tcp			# volume managment server
```
Comment (xTibor) [inline Base/etc/services:395]: I'd rather see this fixed in the IANA registry.

Comment (linusg) [reply Base/etc/services:395]: True - could you point me to a repo or mailing list? Not exactly sure where this is coming from.

Comment (linusg) [reply Base/etc/services:395]: Oops, it says at the top of the file. https://www.iana.org/assignments/service-names-port-numbers/service-names-port-numbers.xhtml

Comment (xTibor) [reply Base/etc/services:395]: I have no idea about their contribution processes. Must be extremely bureaucratic though.

File: Base/usr/share/man/man2/pipe.md [hunk 2]
```diff
@@ -29,7 +29,7 @@ writes some data to the pipe which the parent reads:
 ```c++
 #include <AK/Assertions.h>
 #include <stdio.h>
-#incldue <unistd.h>
+#include <unistd.h>
```
Comment (bugaevc) [inline Base/usr/share/man/man2/pipe.md:32]: 🤦 and here we find out I didn't actually test the examples

**Comment by awesomekling** [comment]
:shirt::shirt::shirt::shirt::shirt::shirt::shirt::shirt::shirt::shirt::shirt:

**Pull request merged**